content/en/docs/how-tos/testing-operator-sdk-operators.md

```diff
@@ -117,140 +117,120 @@ present.
-from the initial operator/bundle to the newly built operator/bundle.
-
-These workflows enhance the simple operator installation workflows (like
-[optional-operators-ci-aws](https://steps.ci.openshift.org/workflow/optional-operators-ci-aws)) with an additional
-[optional-operators-ci-upgrade](https://steps.ci.openshift.org/reference/optional-operators-ci-upgrade) step. Tests
-using these workflows need to provide the parameters specified for the simple operator installation tests plus the following:
-
-|Parameter|Description|
-|:---|:---|
-|`OO_INITIAL_CSV`|The name of initial version of the operator CSV to install.|
-|`OO_INITIAL_CHANNEL`|The name of the channel the initial CSV is in.|
-|`OO_LATEST_CSV`|The name of the CSV being upgraded to.|
+The `OO_BUNDLE` environmental variable set for the step will contain the pull specification of the bundle image.
+Note that the base index can be specified in the [`operator-sdk`](https://sdk.operatorframework.io/docs/cli/operator-sdk_run_bundle/) command.
 
-The following example runs a test that installs an initial version of an operator and then upgrades before performing a functional tests.
 
-`ci-operator` configuration:
+## Operator Upgrade Testing
 {{< highlight yaml >}}
+base_images:
+  my-bundle-init:
+    name: "my-bundle"
+    namespace: my-namespace
+    tag: init
 tests:
-- as: "operator-e2e-upgrade"
+- as: e2e-in-cluster-build
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.12"
   steps:
-    workflow: "optional-operators-ci-aws-upgrade"
-    cluster_profile: "aws"
-    env:
-      OO_CHANNEL: "alpha"
-      OO_INSTALL_NAMESPACE: "myoperator-test-namespace"
-      OO_PACKAGE: "myoperator"
-      OO_TARGET_NAMESPACES: '!install'
-      OO_INITIAL_CSV: "myoperator.v1.1.2"
-      OO_INITIAL_CHANNEL: "v1.1"
-      OO_LATEST_CSV: "myoperator.v1.2.0"
-    dependencies:
-      OO_BUNDLE: "bundle-name" # if the name of the bundle being tested is not my-bundle in the ci-opeator's configuration, update the dependency to match
     test:
-    - as: "e2e"
-      from: "src"               # the end-to-end tests run in the source repository
-      commands: "make test-e2e" # the commands to run end-to-end tests
+    - as: install
+      cli: latest
+      commands: |
+        oc create namespace my-namespace
+        operator-sdk run bundle -n my-namespace "$OO_BUNDLE_INIT" # install my-operator with the version before upgrading
+        oc wait --for condition=Available -n my-namespace deployment my-operator # wait until my-operator is ready
+      dependencies:
+      - env: OO_BUNDLE_INIT
+        name: my-bundle-init
+    - as: upgrade
+      cli: latest
+      commands: |
+        oc create namespace my-namespace
+        operator-sdk run bundle-upgrade -n my-namespace "$OO_BUNDLE" # upgrade my-operator
```

Comment:
@grokspawn 
We have not discussed about upgrade yet.
I found the cmd in the operator-sdk doc.
Are you suggest CI users to use this one to test upgrade?

Reply:
`bundle-upgrade` can absolutely be used for the case where we are presenting a new version of the bundle, for either sqlite/fbc catalogs.  `operator-sdk` will assume that the new version is later than the existing one (and insert as the new head of the channel).

Reply:
Here's the EP for it: https://github.com/operator-framework/enhancements/blob/master/enhancements/sdk-fbc-bundle-upgrade.md